src/main/webapp/script.js

```diff
@@ -88,6 +85,13 @@ function initMap() {
     curNameElement.innerText = place.name;
     curLocElement.innerText = place.geometry.location.toString();
 
-    getRandomRestaurant();
+    var url = new URL('/places-list', window.location.origin),
+        params = {lat: place.geometry.location.lat(), lng: place.geometry.location.lng()}
+    Object.keys(params).forEach(key => url.searchParams.append(key, params[key]))
```

Comment:
What does this line do?

Reply:
This is what the fetch standard currently recommends for URL query parameters
https://fetch.spec.whatwg.org/#fetch-api

Reply:
Interesting, thanks!